snapcraft/plugins/catkin.py

```diff
@@ -66,75 +69,88 @@ def schema(cls):
             },
             'default': [],
         }
+        schema['properties']['source-space'] = {
+            'type': 'string',
+            'default': 'src',
+        }
 
         schema['required'].append('catkin-packages')
 
         return schema
 
     def __init__(self, name, options):
-        super().__init__(name, options)
+        """Sanity-check the plugin config regarding source code paths.
```

Comment:
Is it though? Even the super class only sets a few variables in preparation for more work... doesn't it?